<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connection_to_db_window</class>
 <widget class="QDialog" name="Connection_to_db_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>411</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Connection to database</string>
   </property>
   <widget class="QPushButton" name="connect_db_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Choose Database Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>DB Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Host</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_db_name"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_host"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_port"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_username"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_password"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="test_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>connect_db_button</sender>
   <signal>clicked()</signal>
   <receiver>Connection_to_db_window</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
